doc/release/1.17.0-notes.rst

```diff
@@ -214,16 +226,21 @@ This change is backwards compatible, but now allows code like::
 New Features
 ============
 
-New extensible random module with selectable random number generators
----------------------------------------------------------------------
-A new extensible random module along with four selectable random number
+.. currentmodule:: numpy.random
+
+New extensible `numpy.random` module with selectable random number generators
+-----------------------------------------------------------------------------
+A new extensible `numpy.random` module along with four selectable random number
 generators and improved seeding designed for use in parallel processes has been
-added. The currently available bit generators are MT19937, PCG64, Philox, and
-SFC64. PCG64 is the new default while MT19937 is retained for backwards
+added. The currently available :ref:`Bit Generators <bit_generator>` are
+`~mt19937.MT19937`, `~pcg64.PCG64`, `~philox.Philox`, and `~sfc64.SFC64`.
```

Comment:
Out of curiosity, how does the `~` work?

Reply:
It [removes the leading components](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#xref-syntax) from the link, leaving only the last one. [Here](https://8015-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/release.html#new-features) is the rendered doc